### PR TITLE
Added styled price and type section to dom

### PR DESCRIPTION
**Pull request by MichaelDurfey**

### CODE REVIEW

**Comment by MichaelDurfey** [comment]
Trello ticket: https://trello.com/c/hfwBN68n/32-front-end-refactoring-and-addition-of-price-type-section

File: client/src/components/Price.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  types: PropTypes.arrayOf(PropTypes.string),
+  price: PropTypes.string,
+};
+
+Price.defaultProps = {
+  types: [],
+  price: '',
+};
+
+module.exports = Price;
```
Comment (mattman240) [inline client/src/components/Price.jsx:31]: add blank line if it's not there

File: client/src/components/Stars.jsx [hunk 2]
```diff
@@ -22,7 +22,11 @@ const Stars = ({ numstars }) => {
...
 Stars.propTypes = {
-  numstars: PropTypes.number.isRequired,
+  numstars: PropTypes.number,
+};
+
+Stars.defaultProps = {
+  numstars: 1,
 };
 
 module.exports = Stars;
```
Comment (mattman240) [inline client/src/components/Stars.jsx:32]: same here

**Review by mattman240 — APPROVED** [review]
small request looks good just those two things I commented on. They are small enough that I allow you to merge this request.

**Pull request merged**